deploy/example/cloning/README.md

```diff
@@ -0,0 +1,69 @@
+# Volume Cloning Example
+## Feature Status: Beta
+
+- supported from v1.11.0
+
+## Create a Source PVC
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/csi-driver-smb/master/deploy/example/storageclass-smb.yaml
```

Comment:
remove following command lines since that's for smb server setup, not related to volume cloning
```
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/csi-driver-smb/master/deploy/example/storageclass-smb.yaml
kubectl create secret generic smbcreds --from-literal username=USERNAME --from-literal password="PASSWORD"
kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/csi-driver-smb/master/deploy/example/smb-provisioner/smb-server.yaml
```

Reply:
we still need the storageclass yaml, but does not require smb-provisioner config.
```
kubectl apply -f https://raw.githubusercontent.com/kubernetes-csi/csi-driver-smb/master/deploy/example/storageclass-smb.yaml
```